pkg/entities/defi.go

```diff
@@ -209,17 +209,27 @@ func (e *Entity) InDiscordWalletWithdraw(req request.TransferRequest) (*response
 		return nil, err
 	}
 
-	if _, err := e.HandleUserActivities(&request.HandleUserActivityRequest{
-		GuildID:   req.GuildID,
-		ChannelID: req.ChannelID,
-		UserID:    req.Sender,
-		Timestamp: time.Now(),
-		Action:    req.TransferType,
-	}); err != nil {
-		err = fmt.Errorf("error create activity log: %v", err)
-		return nil, err
+	if req.GuildID == "" {
+		// log activity
+		if err := e.repo.GuildUserActivityLog.CreateOneNoGuild(model.GuildUserActivityLog{
```

Comment:
can we reuse `HandleUserActivities()`?

Reply:
`HandleUserActivities()` contains many logics to handle level up, roles, guild configs, here i only want to log activity